### ISSUE TITLE
Monte Carlo calculation results not making sense

### ISSUE THREAD

**Issue by MaximeAgez** [LCA-ActivityBrowser/activity-browser#821]
Hi,
A colleague of mine performed a Monte Carlo analysis including parameter uncertainty and it yields results that do not make sense.

Explanation:
The stochastic results gives something close to zero:

![image](https://user-images.githubusercontent.com/24285636/182936589-f78f4a54-182f-4471-988d-a22f1ede6f2b.png)

The Monte Carlo including only technosphere uncertainty yields results that make sense:

![image](https://user-images.githubusercontent.com/24285636/182936674-c7f7758c-f0ff-4673-bb05-3996d6ddedb3.png)

Once parameter uncertainty is added it goes off the tracks completely:

![image](https://user-images.githubusercontent.com/24285636/182936837-077b021c-ff42-49fb-97e9-58a292d811c0.png)

You will tell me that there is an issue in the uncertainty definitions for parameters, except I did not yet define any uncertainty for parameters.

![image](https://user-images.githubusercontent.com/24285636/182937057-be03aae8-b691-44f3-832f-b771291f93d2.png)

I will try to run the Monte Carlo analysis in brightway2 and update the post, to see if it's bw2 going crazy of if it's an issue in the implementation in AB.

**Comment by simb-sdu** [comment]
from my understanding, uncertainty can be defined in your parameters as well as in your activity exchanges. 
I had some trouble working modelling with activity parameters, maybe this gives some errors for you as well? It seemed to me that some old parameter data stored somewhere could be mixed and messed totally up. I would suggest to make a brand new activity/product system to test this.

**Comment by MaximeAgez** [comment]
I made a new database in which I only had one activity directly calling an ecoinvent process (i.e., a 1 in the "amount" cell). When performing the Monte Carlo with "technosphere" and "parameters" checked the results look fine.

When I use the formula however (i.e., a 1 in the "formula" cell), Monte Carlo goes crazy with the "parameters" checked.

Since it's the same process and same quantity called it should not have anything to do with remnants of parameters here and there.

**Comment by simb-sdu** [comment]
sounds very strange. I have strange issues with monte carlo too. what happens if you define technosphere uncertainty as "none"? foes it change the results depending on if you put 1 in amount or formula when "none" uncertainty?

Also, what happens if you define "none" uncertainty for all params

**Comment by MaximeAgez** [comment]
Same problems when using "No uncertainty" instead of "Undefined or unknown uncertainty"

**Comment by lucas-sys-cmd** [comment]
I think I met the same problem, too. After including uncertainty in my parameters, the LCA
results were totally different. I've been stuck by this problem for several days.

**Comment by madel91** [comment]
Hello,

I have been struggling with this problem for a long time. I tried many solutions that I can explain to anyone interested as workarounds but in vain. Finally, I had to abandon doing this type of Monte Carlo simulation with the software, and the alternative was developing n number of scenarios= Monte Carlo iterations and doing the random sampling of the parameters manually.

**Comment by haasad** [comment]
could this be cause by the problem described in #1087?

**Comment by gpuigsamper** [comment]
Hello,

I have recently met the same problem when including technosphere and parameter uncertainty in Monte Carlo simulation in AB. Monte Carlo simulation results look fine when excluding parameter uncertainty, but make no sense when including parameter uncertainty.

Do you have any update on this problem?

Thanks.

**Comment by romainsacchi** [comment]
@marc-vdm @mrvisscher I agree that Monte Carlo behaves strangely when given activity parameter uncertainty distributions. Would you happen to have a solution for this?

**Comment by elii-arte** [comment]
Hello I am having this problem too. When I run Monte Carlo with parameter uncertainty only, I get strange results. How can this be fixed?
Is this also happening in BW2?
